graphql/phoneVerification/resolver.ts

```diff
@@ -106,9 +105,13 @@ const phoneVerificationResolver = {
         },
       });
 
-      if (process.env.SMS_VERIFICATION_ENABLED === '1') {
-        const localPhone = getNationalNumber(phone);
-        await sendSms(localPhone, code.toString());
+      const message = `${code} es tu código de verificación CAUDA.`;
+      const localPhone = getNationalNumber(phone);
+      if (process.env.SMS_ENABLED === '1') {
+        sendSms(localPhone, message);
```

Comment:
isn't necessary to await the sensSms function?

Reply:
we are not handling if it fails, that's why I removed the await. Do you think it could break something?